docs/cli/linear.rst

```diff
@@ -0,0 +1,27 @@
+Training and Prediction for Linear Classifiers
+==============================================
+
+Training
+^^^^^^^^
+
+For training, use
+
+.. code-block:: bash
+
+    python3 main.py --config CONFIG_PATH \
+                    --linear [--liblinear_options LIBLINEAR_OPTIONS]
+
+- **config**: configure parameters in a yaml file.
+
```

Comment:
what if validation set specified in config? Mention that validation set isn't used